.changeset/poor-carrots-bathe.md

```diff
@@ -0,0 +1,26 @@
+---
+"miniflare": minor
+---
+
+feat: bind `this` to current `Miniflare` instance in custom service binding handlers
+
+Previously, `this` was unbound in function-valued `serviceBindings`. This change binds it to the current `Miniflare` instance, for easy access to the correct bindings when re-using service functions across instances.
```

Comment:
I assume that these kind of service bindings are very much specific to Miniflare and not generally used by application developers? I ask because I feel just a bit awkward about changing what `this` is in service in general. A more "Workers" idiomatic way to do this would be to provide the Miniflare instance as an `env` binding to the service, no?